src/identity.rs

```diff
@@ -83,29 +83,31 @@ pub trait Identity {
 /// with [with_project_scope](#method.with_project_scope):
 ///
 /// ```rust,no_run
+/// # use osproto::identity::IdOrName;
```

Comment:
Let us reimport it publicly into `osauth::common::IdOrName` to avoid making `osproto` a part of the API.

Reply:
I ended up rather using the already `pub use` statement in `osauth::identity::IdOrName`.